infrastructure/server/oidc_clients_forest_client.tf

```diff
@@ -8,8 +8,8 @@ resource "aws_cognito_user_pool_client" "dev_forest_client_oidc_client" {
     "http://localhost:3000/dashboard",
   ], [for i in range("${var.dev_pr_url_count}") : "https://nr-forest-client-${i}-frontend.apps.silver.devops.gov.bc.ca/dashboard"])
   logout_urls = concat([
-    "${var.cognito_app_client_logout_chain_url.dev}http://localhost:3000/"
-  ], [for i in range("${var.dev_pr_url_count}") : "${var.cognito_app_client_logout_chain_url.dev}https://nr-forest-client-${i}-frontend.apps.silver.devops.gov.bc.ca/"])
+    "http://localhost:3000/logout"
+  ], [for i in range("${var.dev_pr_url_count}") : "https://nr-forest-client-${i}-frontend.apps.silver.devops.gov.bc.ca/logout"])
```

Comment:
remove the logout chain as required, add "/logout" at the end for their logout redirect urls